### PR TITLE
Moveit dependency is failing

### PR DESCRIPTION
**Pull request by BenjaminHug8**
**Description**
Resolve the `moveit` dependency.

### CODE REVIEW

**Comment by BenjaminHug8** [comment]
Waiting on [this issue](https://github.com/ros-planning/moveit2/issues/990).

**Comment by lukicdarkoo** [comment]
Long term, maybe it is better for MoveIt (as well other integrations) to be an optional dependency, or to stress using `webots_ros2_driver` (instead of `webots_ros2`), or to separate `webots_ros2` core packages from examples.

File: scripts/ci_after_init.bash [hunk 1]
```diff
@@ -12,3 +12,5 @@ apt install -y /tmp/webots.deb xvfb
 if [ "${ROS_DISTRO}" = "foxy" ]; then
     apt install -y ros-foxy-turtlebot3-cartographer ros-foxy-turtlebot3-navigation2
 fi
+
+apt-cache policy
```
Comment (omichel) [inline scripts/ci_after_init.bash:16]:
```suggestion
apt-cache policy

```

File: webots_ros2_universal_robot/launch/moveit_demo_launch.py [hunk 2]
```diff
@@ -38,49 +39,63 @@ def load_file(filename):
...
+
+        # Webots simulation with robot
+        launch_description_nodes.append(
+            IncludeLaunchDescription(
+                PythonLaunchDescriptionSource(os.path.join(package_dir, 'launch', 'robot_launch.py'))
+            )
+        )
+    else:
+        launch_description_nodes.append(LogInfo(msg='"moveit" package is not installed, \
+                                                please install it in order to use this demo.'))
```
Comment (omichel) [inline webots_ros2_universal_robot/launch/moveit_demo_launch.py:99]:
```suggestion
                                                please install it in order to run this demo.'))
```

File: webots_ros2_universal_robot/launch/moveit_demo_launch.py [hunk 3]
```diff
@@ -38,49 +39,63 @@ def load_file(filename):
...
-    robot = IncludeLaunchDescription(
-        PythonLaunchDescriptionSource(os.path.join(package_dir, 'launch', 'robot_launch.py'))
-    )
-
-    return LaunchDescription([
-        run_move_group_node,
-        rviz_node,
-        robot
-    ])
+    # Check if nav2_bringup is installed
```
Comment (omichel) [inline webots_ros2_universal_robot/launch/moveit_demo_launch.py:42]: Shouldn't it be moveit here?
```suggestion
    # Check if moveit is installed
```

**Review by omichel — CHANGES_REQUESTED** [review]

File: scripts/ci_after_init.bash [hunk 4]
```diff
@@ -11,6 +11,4 @@ apt install -y /tmp/webots.deb xvfb
 # The following packages are only available in the ROS 2 Foxy distribution. Therefore, we cannot include them in the package.xml, but we have to install them manually here.
 if [ "${ROS_DISTRO}" = "foxy" ]; then
     apt install -y ros-foxy-turtlebot3-cartographer ros-foxy-turtlebot3-navigation2
-fi
-
-apt-cache policy
\ No newline at end of file
+fi
```
Comment (omichel) [inline scripts/ci_after_init.bash:14]: This file is simply missing a carriage return at the end.

**Review by omichel — CHANGES_REQUESTED** [review]

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**